renderer/styles/style.css

```diff
@@ -824,11 +824,13 @@ span.qc-option-txt{
   width: 100%;
   height: 100%;
   font-family: NotoSans;
+  padding: 10px 0 0 0;
```

Comment:
you can define as padding-top: 10px

Reply:
I want to make left, right, and bottom to 0. It is taking some default padding that's why I addedad 10px 0 0 0